src/instrument.ts

```diff
@@ -1,4 +1,5 @@
 import * as Sentry from '@sentry/node';
+// @ts-expect-error this seems like a regression in the types
```

Comment:
I assume you'll follow up on this?

Reply:
Yeah, I'm going to check what happened on the SDK side as it seems related to an upgrade here